mlir/tools/mlir-tblgen/OpPythonBindingGen.cpp

```diff
@@ -1004,8 +1013,8 @@ static void emitValueBuilder(const Operator &op,
                       llvm::join(valueBuilderParams, ", "),
                       llvm::join(opBuilderArgs, ", "),
                       (op.getNumResults() > 1
-                           ? "_Sequence[_ods_ir.OpResult]"
-                           : (op.getNumResults() > 0 ? "_ods_ir.OpResult"
+                           ? "_Sequence[_SubClassValueT]"
```

Comment:
I'm not sure how this works for the type checker given that `_SubClassValueT` is `Type[U]` where `U` is a `TypeVar` that doesn't seem bound?

Reply:
discussed offline